pkg/controller/registry/registry_controller.go

```diff
@@ -180,7 +192,8 @@ func (r *ReconcileRegistry) updateRegistryCR(ctx context.Context, parentInstance
 	}
 
 	// set owner
-	if (originalRegistry.OwnerReferences == nil || len(originalRegistry.OwnerReferences) == 0) && parentInstance != nil {
+	if (originalRegistry.OwnerReferences == nil ||
+		len(originalRegistry.OwnerReferences) == 0) && parentInstance != nil {
```

Comment:
`len(nil)` is `== 0`, anyway

```suggestion
	if (len(originalRegistry.OwnerReferences) == 0 && parentInstance != nil {
```